pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -94,7 +93,7 @@ def _init_feeds_data(self):
         df = self._query_db(
             f"""
                 SELECT contract, pair, timeframe, source
-                FROM {BronzePrediction.get_lake_table_name()}
+                FROM {tbl_parquet_path(self.lake_dir, BronzePrediction)}
```

Comment:
yes, definitely move it to the classes and use BronzePrediction.get_tbl_parquet_name or something like that.

Reply:
moving it there doesn't really help because the Table doesn't know about PPSS and lake dir or exports dir, so I would need to send that as param, and only thing the table will do is since it has the table name just add the '.parquet' at the end.